### ISSUE TITLE
a11y audit, home page

### ISSUE THREAD

**Issue by amygroshek** [Hyperobjekt/untd-site#18]
One lonely issue on the home page pa11y check: 

```
 pa11y https://602d4f4bcc9ec50007022ca8--socialmobility.netlify.app/

Welcome to Pa11y

 > Running Pa11y on URL https://602d4f4bcc9ec50007022ca8--socialmobility.netlify.app/

Results for URL: https://602d4f4bcc9ec50007022ca8--socialmobility.netlify.app/

 • Error: This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name
   ├── #gatsby-focus-wrapper > header > nav > button
   └── <button type="button" class="btn-menu btn btn-link"><svg stroke="currentColor" fill...</button>

1 Errors

**Comment by amygroshek** [comment]
Resolved this today when working on about page merge.
